src/mono/wasm/runtime/dotnet.d.ts

```diff
@@ -151,10 +151,6 @@ type MonoConfig = {
      * If true, the snapshot of runtime's memory will be stored in the browser and used for faster startup next time. Default is false.
      */
     startupMemoryCache?: boolean;
-    /**
-     * hash of assets
-     */
-    assetsHash?: string;
```

Comment:
Where is the hash now, on the API?

Reply:
Does it contain the hash of all the config switches too ?

Reply:
Just assets https://github.com/dotnet/runtime/blob/ea3daec7b379e7ef846c7bb2ae38ffff5a1b597f/src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/GenerateWasmBootJson.cs#L391-L412

Reply:
Is that what we want ? Switches could influence the snapshot, right ?